contents/handbook/engineering/conventions/frontend-coding.md

```diff
@@ -38,7 +38,8 @@ Hence the explicitly in keeping the layers separate.
 - CSS
   - We use regular SCSS files for styling to keep things simple and maintainable in the long run, as opposed to supporting the CSS-in-JS flavour of the month.
   - Inside `MyBlogComponent.tsx` import `MyBlogComponent.scss`
-  - Namespace all your CSS rules under globally unique names like `.my-blog-component { put everything here }`
+  - Namespace all your CSS rules under globally unique classes that match the component's name and case, for example `.DashboardMenu { put everything here }`
+  - We loosely follow BEM conventions. If an element can't be namespaced inside a container class (e.g. modals that break out of the containing DOM element), use BEM style names like `.DashboardMenu__modal` to keep things namespaced.
```

Comment:
I think things are clearer when BEM is used consistently, so I'd also be in favor of codifying it a bit more strongly. Though it's also not a hill I'd want to die on.